### DISCUSSION TITLE
Forward players to a server (BUNGEECORD)

### DISCUSSION THREAD

**Discussion by WWooodyy** [Minestom/Minestom#1082] [Help] [answered]
Took me a quick minute to figure out how to do it due to the ID system based on Spigot, however below is a Util you can use to send players to a server via the Bungeecord Connect Message system.
       
    public static void sendPlayerToServer(Player player, String server) {
        final ByteArrayOutputStream byteStream = new ByteArrayOutputStream();
        final DataOutputStream dataStream = new DataOutputStream(byteStream);

        dataStream.writeUTF("Connect");
        dataStream.writeUTF(server);
        byteStream.close();
        dataStream.close();

        player.getPlayerConnection().sendPacket(new PluginMessagePacket("bungeecord:main", byteStream.toByteArray()));
    }

Usage
`Util#sendPlayerToServer(player, "lobby");`

**Comment by mworzala** [comment, marked as answer]
(see original question)
